src/Frontend/Page/User.hs

```diff
@@ -310,29 +310,29 @@ createdIdeas userId ideasQuery = do
                     (mapM getIdeaStats
                      =<< filter visibleByCurrentUser
                          <$> findIdeasByUserId userId))
-        delegatees <- userDelegateeListsMap userId
+        ds <- delegatees userId
         pure $ PageUserProfileCreatedIdeas
             (setProfileContext user ctx)
             (makeUserView user)
             ideas
-            delegatees)
+            ds)
 
 
--- ** User Profile: User As Delegatee
+-- ** User Profile: User As Delegate
 
 instance ToHtml PageUserProfileUserAsDelegate where
     toHtmlRaw = toHtml
-    toHtml p@(PageUserProfileUserAsDelegate ctx (DeletedUser user) _delegations) = semanticDiv p $ do
+    toHtml p@(PageUserProfileUserAsDelegate ctx (DeletedUser user) _delegationListsMap _delegations) = semanticDiv p $ do
```

Comment:
please rename to `_delegateeListsMap` (or replace with `_`).  or rename the type.